docker-compose.yml

```
@@ -66,6 +67,8 @@ services:
     depends_on:
       - db
       - cache
+    ports:
+      - "${PORT}:3000"
```

@machisuji Shouldn't this fallback:
```suggestion
      - "${PORT:-3000}:3000"
```